pkgs/development/python-modules/matrix-nio/default.nix

```diff
@@ -73,8 +75,11 @@ buildPythonPackage rec {
     hypothesis
     pytest-aiohttp
     pytest-benchmark
+    pytestCheckHook
```

Comment:
why propagate a hook ? it probably should to to buildInputs.

Reply:
I think this is just unfortunate truncation by github. The hook is correctly located in checkInputs